### DISCUSSION TITLE
Indian Host

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#138] [Show and tell]
https://last.gcdn.co/IND***/index.m3u8

[last.gcdn.co.zip](https://github.com/iptv-org/iptv/files/7541958/last.gcdn.co.zip)

I don't even need to give the names. They're given x)

**Comment by SSyar** [comment]
tried around first 25 all were Tamil., some names were not correct

**Reply by BellezaEmporium** [reply]
> 
> 
> tried around first 25 all were Tamil., some names were not correct

I shall assume it's a feedback issue from the server. I've began compiling some names, and found out part of them were from YuppTV, other parts were from Zee5.

**Comment by SSyar** [comment]
![image](https://user-images.githubusercontent.com/17178080/142623320-8d906a30-4ed8-4cce-9394-58d4fcbe24c7.png)
![image](https://user-images.githubusercontent.com/17178080/142623400-dacf106a-0392-44cd-8eec-d76785a84cf6.png)
![image](https://user-images.githubusercontent.com/17178080/142623488-08bc4e04-c7b4-45fe-bcf8-6fc7f56fafd1.png)
![image](https://user-images.githubusercontent.com/17178080/142623608-b2e0e8c0-2d46-4a5a-90df-a4a7d059d11d.png)
![image](https://user-images.githubusercontent.com/17178080/142623793-00bcc168-d769-4964-a114-5ffa81e1d0bf.png)
![image](https://user-images.githubusercontent.com/17178080/142623850-a620f6b5-365a-4bbd-967c-1dafd6475360.png)

**Reply by BellezaEmporium** [reply]
You had reported that some of those channels were badly named. Which ones are they ?

**Reply by SSyar** [reply]
289 was not CN which i remember

**Reply by BellezaEmporium** [reply]
I see. I'll check.

EDIT : Confirmed being D TAMIL (https://www.exchange4media.com/media-tv-news/d-d-tamil-to-unveil-new-logo-with-actress-tamannaah-bhatia-112293.html) / https://toolswift.com/tv-schedule/tamil/d-tamil
